pkg/webui/console/components/rights-group/index.js

```diff
@@ -18,22 +18,27 @@ import { defineMessages } from 'react-intl'
 import classnames from 'classnames'
 
 import PropTypes from '../../../lib/prop-types'
+import { RIGHT_ALL } from '../../lib/rights'
+
```

Comment:
redundant new line

Reply:
No, we separate `lib` from component imports in other places too.

Reply:
no we dont, but whatever

Reply:
After looking at it again, let's say there is at least no consistency in how we do it, we do however indeed not separate them in most cases.

https://github.com/TheThingsNetwork/lorawan-stack/blob/master/pkg/webui/console/containers/webhooks-table/index.js

https://github.com/TheThingsNetwork/lorawan-stack/blob/master/pkg/webui/components/location-form/index.js

I guess this is a case for clarification in some yet to be written coding conventions.